### PR TITLE
ci: add contribution guide

### PR DESCRIPTION
**Pull request by Oppen**
## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [x] Documentation has been added/updated.
  - [ ] CHANGELOG has been updated.

### CODE REVIEW

**Review by fmoletta — APPROVED** [review]
Looks great!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+-->
+- [Styleguides](#styleguides)
+- [Commit Messages](#commit-messages)
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation](docs/).
+
+Before you ask a question, it is best to search for existing [Issues](/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
```
Comment (fmoletta) [inline CONTRIBUTING.md:32]: The issues link may be broken

**Review by pefontana — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (pefontana) [reply CONTRIBUTING.md:32]: maybe is better to link `https://github.com/lambdaclass/cairo-rs/issues` ?

Comment (Oppen) [reply CONTRIBUTING.md:32]: It's relative to the repository, so it should work. I can test it to be sure.

Comment (Oppen) [reply CONTRIBUTING.md]: Huh, it was indeed broken, nice catch!

Comment (Oppen) [reply CONTRIBUTING.md]: Now all links work.

**Pull request merged**